\documentclass[a4paper,9pt]{article}

\usepackage[english]{babel}
\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\usepackage{amsmath,amssymb}
\usepackage{subcaption}
\usepackage{hyperref, url}
 \usepackage[table,xcdraw]{xcolor}
\parindent 0mm
\parskip 3mm

% add your name and student number in parenthesis
\title{Survival Analysis of TCGA Liver cancer expressed gene transcript }
\author{ Sailendra Pradhananga (sailendra.pradhanaga@scilifelab.se)\\ \\
			Project Report:Algorithms in Bioinformatics}

\begin{document}

\maketitle

\section{Introduction}


Survival analysis are statistical toolkits that are used to analyzing the outcome of variables based on time period of event occurrence. The event could be either be death, outbreak of disease or failure of automobile parts which are then followed specified time period in days, weeks or years. The major part of the these type of data involves two kinds of variables: first is the time to event and second whether the event have occurred or not. Based on different methods such as non parametric model such Kaplan Meier method or regression model such as cox proportional hazards two function, survival function and hazard function. The survival function gives the probability  for each time-points of surviving up to that events while hazard function whether the event has occurred or not  given the individual has survived or not. The main aim of these type of analysis is find the relationship between the survival time and variables under study such as drugs, gene expression, treatment  either with different covariates such as age, gender, race.

In the current project, we are provided a liver cancer patients of different races  dataset survival time and gene expression dataset of various grades. The main aim was to model the data with survival model in order find the potential prognostic marker for survival of patients using Cox Proportional Hazard models. Additionally, using the multiple hypothesis correction we predicted 4698 gene transcripts at 0.05 FDR. 

\section{Method}

	\subsection{TCGA Dataset}
The  TCGA dataset for the analysis was download from the (http://kaell.org/files/survivalLIHC.txt) site. As already stated in the project guidelines, these are the cancer patient expression data from the sequenced liver tumor.  Additionally there are few other features implication each cancer patient ethnicity, sex and cancer types.  These data were first read into python notebook as pandas data frame  as done in  jupyter notebook.  Additional exploratory analysis was done in the notebook itself. The  dataset consists of both categorical features and numerical data of gene expression dataset. 

	\subsection {Preprocessing of Dataset}
In the survival analysis, the different categorical features were provided of liver cancer patients. These categorical features includes Gender, Race, Stage , Status, Age and LivingDays of different cancer patients.  These categorical features were then coded into different numerical codes . Originally, the dataset consists of gene expression profile os $19,571$. Furthermore, only those gene expression data were taken which had at least an expression in one of the patients.  

	\subsection{Cox proportional Hazard model}
	 Cox proportional hazards regression, used to relate several risk factors or exposures, considered simultaneously, to survival time. In the current project we used the implementation of Cox regression hazard model implemented on lifelines packages. Although I had tried including other covariates such as age and temperature to be used in the model, at this phase we have used individual expression of each gene transcript iteratively in order to get hazard function and survival rates of each gene transcripts. Based on proportional hazard model we get an p-value estimates of gene transcript. However not all the gene transcripts fitted into the Cox survival model thus we excluded to get final list of pvalues for each transcripts.
	 
	\subsection {Multiple testing correction}
	As discussed in earlier exercises we have  implemented multiple testing correction for all the p.value estimates of gene expression for liver cancer to occur. However since we are testing multiple features, the pvalue $<$ 0.05 might be too lenient estimates. Hence based on storey and tribasni papser \cite {} an q value estimates were made of all the pvalues calculated. We fitted the cubic spline in the $\pi_0$ estimates in the given  $\lambda$ values.
	 


 \section{Result}
 
 	\subsection {TCGA Liver cancer characteristics}
 
 The data consists of 6 categorical features as shown in table 1 . Additionally since we are interested to know the dependent variable of event that in our case alive and death as a function of Living day we can observe the marked difference of alive and death events in patient cohort .  Additionally, there is markedly difference in gender distribution  in cancer cohort with more than half of the patient are male as shown in fig 2  and table 1. 

 

\begin{figure}[bh]
    \centering
      \includegraphics[width=\textwidth]{Genderandstatus.png}
    \caption{Pictures of animals}\label{fig:animals}
\end{figure}



\begin{table}[bh]
\centering
\scalebox{0.5}{
	\label{my-label}
	\begin{tabular}{|l|l|}
	\hline
		\multicolumn{2}{|l|}{\cellcolor[HTML]{96FFFB}{\color[HTML]{FE0000} Summary statistics of TCGA liver cancer cohorts}}                                                                                                                                 	\\ \hline
		\multicolumn{2}{|c|}{Gender}                                                                                                                                                                                                                         	\\ \hline
			\begin{tabular}[c]{@{}l@{}}male\\ \\ female\end{tabular}                                                                                                       & \begin{tabular}[c]{@{}l@{}}245\\ \\ 		119\end{tabular}                               \\ \hline
		\multicolumn{2}{|c|}{Race}                                                                                                                                                                                                                           \\ \hline
			\begin{tabular}[c]{@{}l@{}}white\\ asian\\ black or african american\\ not reported\\ american indian or alaska native\end{tabular}                             & \begin{tabular}[c]{@{}l@{}}182\\ 154\\ 17\\ 10\\ 1\end{tabular}                    \\ \hline
		\multicolumn{2}{|c|}{Liver cancer stage}                                                                                                                                                                                                             \\ \hline
			\begin{tabular}[c]{@{}l@{}}stage i\\ stage ii\\ stage iiia\\ not reported\\ stage iiic\\ stage iiib\\ stage iii\\ stage ivb\\ stage iv\\ stage iva\end{tabular} & \begin{tabular}[c]{@{}l@{}}170\\ 83\\ 63\\ 24\\ 9\\ 8\\ 3\\ 2\\ 1\\ 1\end{tabular} \\ \hline
		\multicolumn{2}{|c|}{Status}                                                                                                                                                                                                                         \\ \hline
		\begin{tabular}[c]{@{}l@{}}alive\\ dead\end{tabular}                                                                                                            & \begin{tabular}[c]{@{}l@{}}234\\ 130\end{tabular}                                  \\ \hline
	\end{tabular}}
\caption{Pictures of animals}\label{fig:animals}
\end{table}

 
 	\subsection {Survival analysis of dataset}
	
	As discussed earlier , the cox proportional hazard survival analysis gives us estimates of survival probabilities and cumulative hazard  of each transcript. As shown in the figure 2  for one of the transcripts.. The model yields the p-values for significance of the regression model.  There were $6032$ features significant at pvalue $< 0.05$  which is quite alot. Thus in order to control the multipe testing of features , we need  used the multiple hypothesis correction as indicated in Storey and tibsachi paper. 

\begin{figure}[bh]
    \centering
      \includegraphics[width=0.75\textwidth]{survial_plot_one_mock_gene.png}
    \caption{Pictures of animals}\label{fig:animals}
\end{figure}
  	
	
	\subsection {Estimating $\lambda$ estimates on pvalues calculated}
\begin{figure}[h]
    \centering
      \includegraphics[width=\textwidth]{lambdaestimatescurvefitting.png}
    \caption{Pictures of animals}\label{fig:animals}
\end{figure}


	\subsection {Estimation of expression levels based on FDR}
\begin{figure}[h]
    \centering
      \includegraphics[width=\textwidth]{comb.png}
    \caption{Pictures of animals}\label{fig:animals}
\end{figure}

\bibliographystyle{unsrt}
\bibliography{lib}

\end{document}